docs/tools/libnvme.md

```diff
@@ -6,27 +6,27 @@ sidebar_label: libnvme User Library
 
 # libnvme User Library
 
-*libnvme* is an open source user library providing defintions and functions for
-interacting with nvme devices. While nvme-cli provides convenient ways for a
-user to interact with nvme devices from the shell, libnvme provides similiar
-access for other programs.
+*libnvme* is an open source user library that provides defintions and functions
+for interacting with nvme devices. `nvme-cli` provides convenient ways for a
+user to interact with nvme devices from the shell, and libnvme provides
+similiar access for other programs.
 
 ## Overview
 
-*libnvme* provides functions for discoverying and managing all nvme devices in
-a linux environment. When the NVMe ZNS specification was ratified, libnvme
-incorporated defintions for all the types and commands that specification
-provides.
+*libnvme* provides functions for discovering and managing all nvme devices in a
+Linux environment. When the NVMe ZNS specification was ratified, libnvme
+incorporated defintions for all the types and commands that the NVMe ZNS
```

Comment:
s/defintions/definitions

Reply:
Serves me right for editing while tired.